RetroBar/PropertiesWindow.xaml

```diff
@@ -229,6 +229,15 @@
                                       IsChecked="{Binding Source={x:Static Settings:Settings.Instance}, Path=ShowTaskThumbnails, UpdateSourceTrigger=PropertyChanged}">
                                 <Label Content="{DynamicResource show_window_previews}" />
                             </CheckBox>
+                            <DockPanel>
+                                <Label VerticalAlignment="Center"
+                                       Target="{Binding ElementName=cboEdgeSelect}">
+                                    <AccessText Text="{DynamicResource rowcount_text}"
+                                                ToolTip="{DynamicResource rowcount_tip}" />
+                                </Label>
+                                <ComboBox ItemsSource="{DynamicResource rowcount_options}"
+                                          SelectedValue="{Binding Source={x:Static Settings:Settings.Instance}, Path=RowCount, UpdateSourceTrigger=PropertyChanged}" />
+                            </DockPanel>
```

Comment:
1. Since this does not affect vertical taskbars, lets disable it when the the Location is left or right.
2. I think this would look better placed below the Location combobox.

Reply:
Do you want it disabled or hidden?

Reply:
Let's keep it disabled in order to prevent the window changing size.